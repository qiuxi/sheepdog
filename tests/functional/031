#!/bin/bash

# Test vdi list after cluster restart

. ./common

for i in `seq 0 5`;do _start_sheep $i;done
_wait_for_sheep 6

_cluster_format -c 1

for i in 1 2 3; do
    $DOG vdi create test$i ${i}00M -c $i
done

$DOG vdi create -e test4 400M

_vdi_list

$DOG cluster shutdown
_wait_for_sheep_stop

for i in `seq 0 5`;do _start_sheep $i;done
_wait_for_sheep 6

_vdi_list
