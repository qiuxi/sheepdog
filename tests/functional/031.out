QA output created by 031
using backend plain store
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
  test4        0  400 MB  0.0 MB  0.0 MB DATE   fd2de3     6              
  test1        0  100 MB  0.0 MB  0.0 MB DATE   fd32fc     1              
  test3        0  300 MB  0.0 MB  0.0 MB DATE   fd3662     3              
  test2        0  200 MB  0.0 MB  0.0 MB DATE   fd3815     2              
  Name        Id    Size    Used  Shared    Creation time   VDI id  Copies  Tag
  test4        0  400 MB  0.0 MB  0.0 MB DATE   fd2de3     6              
  test1        0  100 MB  0.0 MB  0.0 MB DATE   fd32fc     1              
  test3        0  300 MB  0.0 MB  0.0 MB DATE   fd3662     3              
  test2        0  200 MB  0.0 MB  0.0 MB DATE   fd3815     2              
